scripts/setup/install_aws_dependencies.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+#
+# MIT License
+#
+# Copyright (c) 2023 EASL and the vHive community
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+LOADER_NODE=$1
+GO_VERSION=1.20.4
+
+server_exec() {
+  ssh -oStrictHostKeyChecking=no -p 22 $LOADER_NODE $1;
+}
+
+# Check the architecture
+arch=`server_exec 'uname -m'`
+case $arch in
+  'x86_64')
+    arch='amd64'
+    ;;
+  'aarch64')
+    arch='arm64'
+    ;;
+  *)
+    echo "Unsupported architecture $arch"
+    exit 1
+    ;;
+esac
+
+echo "Installing the dependencies for AWS deployment"
+
+# ========== Get loader ==========
+server_exec "git clone --branch=main https://github.com/vhive-serverless/invitro.git loader"
```

Comment:
It's not necesserily main branch. You can use the one that set in setup.cfg in the same way as create_multinode.sh uses.